ludwig/model_export/coreml_exporter.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC, abstractmethod
+
+import coremltools as ct
+import torch
+from model import MyNet
+
+from ludwig.api import LudwigModel
+from ludwig.model_export.base_model_exporter import BaseModelExporter
+
+ class CoreMLExporter(BaseModelExporter):
+    def export_classifier(self, model_path, export_path):
+        ludwig_model = LudwigModel.load(model_path)
+        model = CoreMLExporter(ludwig_model.model)  # Wrap the model
+        #model.eval()  # inference mode, is this needed.. I think onnx export does this for us
+
+        width = ludwig_model.config["input_features"][0]["preprocessing"]["width"]
+        height = ludwig_model.config["input_features"][0]["preprocessing"]["height"]
+        example_input = torch.randn(1, 3, width, height, requires_grad=True)
+
+        # Create the model and load the weights
+        model = MyNet()
+        model.load_state_dict(torch.load(model_path))
+        # Create dummy input
+        dummy_input = torch.rand(1, 3, 32, 32)
+        # Trace the model
+        traced_model = torch.jit.trace(model, dummy_input)
+        # Create the input image type
+        input_image = ct.ImageType(name=example_input, shape=(1, 3, 32, 32), scale=1/255)
```

Comment:
Nit: can we do `scale=1 / 255.0` to ensure that it is a float.

Also, I thought example input has `shape=(1, 3, width, height)`. Where did `32` come from?

Reply:
fixed